corehq/apps/export/views.py

```diff
@@ -872,6 +872,14 @@ def get_filters(self, filter_form_data):
         filter_form = self._get_filter_form(filter_form_data)
         return filter_form.get_case_filter()
 
+    def _get_filter_form(self, filter_form_data):
+        filter_form = self.filter_form_class(
+            self.domain_object, filter_form_data
```

Comment:
Yeah I think you can merge now. Looks like the test already failed on `test_compress_offline`, which I don't think is related.
(you can see that failure if you view the build on travis, not sure why it doesn't show on the PR yet)